catalog/spatial/geocoding/spatial-geocoding-feature/src/test/java/org/codice/ddf/spatial/geocoding/feature/TestCatalogFeatureIndexer.java

```diff
@@ -188,7 +199,13 @@ private Metacard getExampleMetacard() {
     WKTWriter writer = new WKTWriter();
     String wkt = writer.write((Geometry) getExampleFeature().getDefaultGeometry());
     metacard.setAttribute(new AttributeImpl(Core.LOCATION, wkt));
-    metacard.setAttribute(new AttributeImpl(Core.TITLE, COUNTRY_CODE));
+    metacard.setAttribute(new AttributeImpl(Core.TITLE, TITLE));
+    metacard.setAttribute(new AttributeImpl(Location.COUNTRY_CODE, COUNTRY_CODE));
     return metacard;
   }
+
+  private MetacardType generateMetacardType() {
+    return new MetacardTypeImpl(
```

Comment:
Should this be replaced with our standard metacardtypeimpl from the testing libraries?

Reply:
What testing library?

Reply:
I was thinking about https://github.com/connexta/gsr-yorktown/blob/master/catalog/core/common/catalog-core-common-xmlutils/src/test/java/mil/nga/gsr/catalog/xml/utils/InputTransformerTest.java#L41, but that would have to be refactored out of InputTransformerTest